src/mibase.ts

```diff
@@ -436,64 +437,74 @@ export class MI2DebugSession extends DebugSession {
 		};
 
 		if (id instanceof VariableScope) {
-			let stack: Variable[];
 			try {
-				stack = await this.miDebugger.getStackVariables(id.threadId, id.level);
-				for (const variable of stack) {
-					if (this.useVarObjects) {
-						try {
-							const varObjName = VariableScope.variableName(args.variablesReference, variable.name);
-							let varObj: VariableObject;
+				if (id.name != "Registers") {
+					const stack: Variable[] = await this.miDebugger.getRegisters();
```

Comment:
why is this using `getRegisters` inside `id.name != "Registers"`?

Reply:
Fixed.